nss-day-cohort-52/rare-api-smoky-pythons
# Lz reactions

### Issue ticket(s): #38 #39 

### New file(s): none

### Modified File(s): views/post.py and models/post.py

### Modifications: Added @action methods to the views/post.py and an @action property to the models/post.py 

### Steps to test:
1. While in the terminal, use git fetch --all in the root directory of this repo
2. Checkout out to lz-reactions
3. Run the server. 
4. Check to see if you can make a new reaction. 
5. See if you can reaction a post and see the number update on the DOM.
